package.json

```diff
@@ -6,6 +6,7 @@
     "deep-equal": "1.0.1",
     "harmony-reflect": "1.5.1",
     "isomorphic-fetch": "2.2.1",
+    "key-mirror": "1.0.1",
```

Comment:
if we are going to use this dependency instead of the `utils` key_mirror, shouldn't we update all the references to it in the constant and action types files?